db/migrate/20221018195116_create_like.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
```

Comment:
- [ ] there should be a reference to Author and post in the like table in the form of a foreign key.